kusto/conn.go

```diff
@@ -218,6 +207,30 @@ func (c *conn) doRequest(ctx context.Context, execType int, db string, query Stm
 	return op, header, resp.Header, body, nil
 }
 
+func (c *conn) getHeaders(properties requestProperties) http.Header {
+	header := http.Header{}
+	header.Add("Accept", "application/json")
+	header.Add("Accept-Encoding", "gzip")
+	header.Add("Content-Type", "application/json; charset=utf-8")
+	header.Add("x-ms-version", "2019-02-13")
+	header.Add("x-ms-client-request-id", "KGC.execute;"+uuid.New().String())
```

Comment:
Don't we support user provided ClientRequestId in Go?

Reply:
Nope, but I'll add